project/home/models.py

```diff
@@ -11,3 +15,47 @@ class HomePage(Page):
     content_panels = Page.content_panels + [
         FieldPanel('intro', classname="full"),
     ]
+
+    def get_context(self,request):
+        context = super().get_context(request)
+        context['invite_friends'] = self.get_children().type(InviteFriendsPage)
```

Comment:
If we set `max_count = 1` on the `InviteFriendsPage` model, we can get the _only_ instance using `InviteFriendsPage.objects.get()`
```suggestion
        context['invite_friends_page'] = InviteFriendsPage.objects.get()
```